include/seastar/core/preempt.hh

```diff
@@ -34,12 +34,10 @@ struct preemption_monitor {
     std::atomic<uint32_t> tail;
 };
 
-}
-
-extern __thread const internal::preemption_monitor* g_need_preempt;
+extern __thread const preemption_monitor* g_need_preempt;
```

Comment:
```suggestion
}

extern __thread const internal::preemption_monitor* g_need_preempt;

namespace internal {
```


@rzarzynski sorry for the bike shedding . i was wrong. this variable should be in `seastar` namespace, even though it's should have been in `seastar::internal`..